src/app/[locale]/(business)/(with-header)/friend-requests/actions.ts

```
@@ -12,53 +12,61 @@ import {
 } from "@/domain/users/errors";
 import { getCurrentUser } from "@/lib/auth";
```

ESLint

[error] There should be no empty line within import group

(import/order)